### PR TITLE
FEAT: playlist management

### PR DESCRIPTION
**Pull request by partha8**
# Files to check
1. [src/components/PlaylistModal/PlaylistModal.js](https://github.com/partha8/weeb-culture/compare/playlist-management?expand=1#diff-e075c7fe32c9bdcb7b999668a499af2373cbad0e14fb124ff48ecbb766488fbc)
2. [src/components/PlaylistModal/playlist-modal.css](https://github.com/partha8/weeb-culture/compare/playlist-management?expand=1#diff-13723c227368434775a50322b8a355c35f452f0190357d46a48df21037680cc4)
3. [src/hooks/useGetPlaylists.js](https://github.com/partha8/weeb-culture/compare/playlist-management?expand=1#diff-4588b314d814b13b5006d71407b1d0ce853fb89f5d9283e58ef4ddc7b4c85d60)
4. [src/pages/Playlist/Playlist.js](https://github.com/partha8/weeb-culture/compare/playlist-management?expand=1#diff-fe11b350047c16e188202afe34b533323ba1a221eb0d203a1b7b6a13bf7df73d)
5. [src/pages/PlaylistsContainer/PlaylistsContainer.js](https://github.com/partha8/weeb-culture/compare/playlist-management?expand=1#diff-fbaae490125923e786b01746df3ad01abedf4807bb0ed33aa5030cc5610634ac)
6. [src/reducers/videosReducer.js](https://github.com/partha8/weeb-culture/compare/playlist-management?expand=1#diff-9f9340b9067e5f2155aea27f29aa2d223cd62a977fac23a99b7c0f605ff6680a)
7. [src/utils/playlistUtils.js](https://github.com/partha8/weeb-culture/compare/playlist-management?expand=1#diff-0aab89e138f475dd3aa1d8a3df2c7e153b655e024c03d19c6bffe28b9c5315b6)

# Future Improvements
- Have different side-panel videos for different pages and not show "must-watch" videos only.
- Better UI for different pages

### CODE REVIEW

**Comment by Dev-AkshatGupta** [comment]
Nothing to suggest you everything seems good to me. Not a single thing to suggest great work

**Review by SukanyaSen530 — APPROVED** [review]

File: src/utils/playlistUtils.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+};
+
+export const addVideoToPlaylist = async (
+  selectedVideo,
+  videosDispatch,
+  playlistId,
+  toastHandler
+) => {
+  const encodedToken = localStorage.getItem("token");
+  console.log("inside add");
```
Comment (abhisheksahani-nova) [inline src/utils/playlistUtils.js]: Remove this console, if not needed in feature

**Review by abhisheksahani-nova — APPROVED** [review]
Everything looks good and clean , good to merge

**Comment by partha8** [comment]
Thank you @Dev-AkshatGupta, @SukanyaSen530 and @abhisheksahani-nova for the review.

**Pull request merged**